book/website/community-handbook/local-build.md

```diff
@@ -54,10 +54,21 @@ $ python3 -m venv ./venv
 ```
 
 Next, active the virtual environment,
-
+::::{tab-set}
+:::{tab-item} Linux
```

Comment:
Maybe call this Unix so it covers both Linux and Mac since the same command works on both? (Nitpicky, I admit!)